pkg/chrootarchive/archive_windows.go

```diff
@@ -12,7 +12,7 @@ func chroot(path string) error {
 	return nil
 }
 
-func invokeUnpack(decompressedArchive io.ReadCloser,
+func invokeUnpack(decompressedArchive io.Reader,
```

Comment:
could someone tell why do I have to make this change to pass PR jenkins auto build?
the windows is out of my interest and doesn't belong to this PR.